build_WOFF.py

```diff
@@ -0,0 +1,20 @@
+import glob, os
+from fontTools import ttLib
+
+
+os.chdir("build")
+
+for file in glob.glob("*.ttf"):
```

Comment:
You could use pathlib.Path here, simplifies the code somewhat:
```python
from pathlib import Path

for file in Path(".").glob("*.ttf"):
    ttfile = ttLib.TTFont(file)

    if "TSI0" in ttfile:
        print(f"Warning! VTT production files present in font: {file}")
        print("Please ship the font out of VTT before converting")
        print("*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***")
        sys.exit(1)

    print(f"{file}: Generating WOFF2")
    ttfile.flavor = "woff2"
    ttfile.save(file.with_suffix(".woff2))
    print("All done")
    print("*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***") 
```

Reply:
Looks good!

Reply:
(I'm curious: this comment is marked resolved, but the script still isn't using pathlib. Shall we file a followup workitem to clean it up? :smile:)

Reply:
Merged it in (and modified a bit).